File: src/StudentManager/StudentManager.Service/Models/Users/TeacherDto.cs
namespace StudentManager.Service.Models.Users;

public record TeacherDto
{
    public string Id { get; init; }
    public string TelegramId { get; init; }
    public string? Email { get; init; }

    public string FirstName { get; init; }
    public string LastName { get; init; }
    public string? Patronymic { get; init; }
}

File: src/StudentManagerBot/src/main/java/orazzu/studentmanagerbot/dto/TeacherDto.java
package orazzu.studentmanagerbot.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@NoArgsConstructor
@AllArgsConstructor
@Data
public class TeacherDto {
    private Long tgId;
    private String telegramUsername;
    private String id;
    private String email;
    private String firstName;
    private String lastName;
    private String patronymic;
}
